lib/trento/infrastructure/software_updates/suma.ex

```diff
@@ -115,4 +119,14 @@ defmodule Trento.Infrastructure.SoftwareUpdates.Suma do
        }}
     end
   end
+
+  defp maybe_write_ca_cert_file(nil), do: File.rm(SumaApi.ca_cert_path())
+
+  defp maybe_write_ca_cert_file(ca_cert) do
+    SumaApi.ca_cert_path()
+    |> Path.dirname()
+    |> File.mkdir_p!()
+
+    File.write(SumaApi.ca_cert_path(), ca_cert)
+  end
```

Comment:
To me `File.rm` and `File.write` are two different forms of writing, so this can just be names `write`, without the `maybe`

```suggestion
  defp write_ca_cert_file(nil), do: File.rm(SumaApi.ca_cert_path())

  defp write_ca_cert_file(ca_cert) do
    SumaApi.ca_cert_path()
    |> Path.dirname()
    |> File.mkdir_p!()

    File.write(SumaApi.ca_cert_path(), ca_cert)
  end
```